app/templates/brief.html

```diff
@@ -43,14 +43,19 @@ <h2>This opportunity is closed for applications.</h2></p>
         <h2>
           Preview only
         </h2>
-        <p class="banner-message">
-          This brief is not published yet. {% if brief_of_current_user %}
+        <p class="banner-message"> 
+          <p>This brief is not yet published.</p> {% if unanswered_required > 0 %}
```

Comment:
Should this also check for `brief_of_current_user` ?

Reply:
good pickup